### ISSUE TITLE
Shuffle the examples in prediction phase.

### ISSUE THREAD

**Issue by woshiyyya** [zihangdai/xlnet#28]
I notice that in [run_classifier.py](https://github.com/zihangdai/xlnet/blob/master/run_classifier.py), row 409, your code shuffles the input examples. 

While in predicting phase, we actually need an ordered input sequence for online submission. 

You may add a flag here.

```python
tf.logging.info("Create new tfrecord {}.".format(output_file))
writer = tf.python_io.TFRecordWriter(output_file)

## Here
np.random.shuffle(examples)

if num_passes > 1:
    examples *= num_passes
```

**Comment by zihangdai** [comment]
Thank you for pointing out this. We will fix this later today.

**Comment by kimiyoung** [comment]
Fixed in https://github.com/zihangdai/xlnet/commit/8f17cfd59963a2c68fc76b00ac8f1d66e50b09ee

**Comment by pluieciel** [comment]
I still get shuffled result from predict. So I need to add the "guid" to help me recognize the examples.
Could anyone tell how to get an ordered output in predicting phase?

Thanks a lot!

Output:
I0724 16:17:42.169244 140173828224896 run_classifier.py:830] Predicting submission for example: 0
\*******+++Predicting submission for example: 0 {'guid': 268, 'logits': array([-7.6827693,  7.1939244], dtype=float32), 'labels': 0, 'is_real': 1}
I0724 16:17:42.170245 140173828224896 run_classifier.py:830] Predicting submission for example: 1
\*******+++Predicting submission for example: 1 {'guid': 152, 'logits': array([ 2.7317226, -1.8245108], dtype=float32), 'labels': 0, 'is_real': 1}
I0724 16:17:42.170711 140173828224896 run_classifier.py:830] Predicting submission for example: 2
\*******+++Predicting submission for example: 2 {'guid': 447, 'logits': array([-7.67181 ,  7.545555], dtype=float32), 'labels': 0, 'is_real': 1}
I0724 16:17:42.171119 140173828224896 run_classifier.py:830] Predicting submission for example: 3
\*******+++Predicting submission for example: 3 {'guid': 263, 'logits': array([-7.490347,  7.14594 ], dtype=float32), 'labels': 0, 'is_real': 1}
I0724 16:17:42.171572 140173828224896 run_classifier.py:830] Predicting submission for example: 4
\*******+++Predicting submission for example: 4 {'guid': 365, 'logits': array([-6.964435,  6.12909 ], dtype=float32), 'labels': 0, 'is_real': 1}
